packages/core/stylus/package.json

```
@@ -28,4 +29,4 @@
     "typescript": "^5.0.0",
     "semver": "^7.6.0"
   }
-}
+}
```

```suggestion
}

```
nit: missing newline char